composer.json

```diff
@@ -31,14 +31,15 @@
         "php": ">=7.1.0",
         "ext-intl": "*",
         "ext-mbstring": "*",
-        "cakephp/chronos": "^1.0.1",
         "aura/intl": "^3.0.0",
+        "cakephp/chronos": "^1.0.1",
+        "mikey179/vfsStream": "^1.6",
```

Comment:
`vfsStream` should be in `require-dev` as in https://github.com/cakephp/cakephp/pull/11931

Reply:
Yup, my bad, will fix it.